example/bank/cmd/api/main.go

```diff
@@ -0,0 +1,104 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"net/http"
+	"os"
+	"os/signal"
+
+	"github.com/go-chi/chi"
+	"github.com/go-chi/chi/middleware"
+	"github.com/hellofresh/goengine/example/bank/config"
+	"github.com/hellofresh/goengine/example/bank/domain"
+	goengineZap "github.com/hellofresh/goengine/extension/zap"
+	"go.uber.org/zap"
+)
+
+func main() {
+	logger, err := zap.NewDevelopment()
+	failOnError(err)
+
+	db, dbCloser, err := config.NewPostgresDB(logger)
+	failOnError(err)
+	defer dbCloser()
+
+	manager, err := config.NewGoEngineManager(db, goengineZap.Wrap(logger))
+	failOnError(err)
+	failOnError(config.SetupDB(manager, db, logger))
+
+	eventStore, err := manager.NewEventStore()
+	failOnError(err)
+
+	bankAccountRepo, err := domain.NewBankAccountRepository(eventStore, config.EventStoreStreamName)
+	failOnError(err)
+
+	router := chi.NewRouter()
+	router.Get("/", listRoutes(router, logger))
+	router.Get("/report/", bankReportsHandler(db, logger))
+	router.Mount("/debug", middleware.Profiler())
+	router.Mount("/account", newAccountHandler(db, bankAccountRepo, logger).HTTPHandler())
```

Comment:
This one stands a little bit out. Why don't you return `*chi.Mux` straight away instead of calling `newAccountHandler(...). HTTPHandler()`?